scripts/setup_networking.py

```diff
@@ -5,76 +5,89 @@
 # wlan0 and ap0 interfaces. This configuration allows switching between
 # wireless client mode (wlan0) and wireless access point mode (ap0)
 
-import os
 import subprocess
 
-print("[ INSTALLING SYSTEM REQUIREMENTS ]")
-subprocess.call(["apt", "install", "libnss-resolve"])
 
-print("[ DEINSTALLING CLASSIC NETWORKING ]")
-subprocess.call(["apt",
-                 "--autoremove",
-                 "purge",
-                 "ifupdown",
-                 "dhcpcd5",
-                 "isc-dhcp-client",
-                 "isc-dhcp-common",
-                 "rsyslog"])
-subprocess.call(["apt-mark",
-                 "hold",
-                 "ifupdown",
-                 "dhcpcd5",
-                 "isc-dhcp-client",
-                 "isc-dhcp-common",
-                 "rsyslog",
-                 "openresolv"])
-subprocess.call(["rm", "-r", "/etc/network", "/etc/dhcp"])
+def configure_networking():
+    
+    print("[ INSTALLING SYSTEM REQUIREMENTS ]")
+    subprocess.call(["apt", "install", "libnss-resolve"])
 
-print("[ SETTING UP SYSTEMD-RESOLVED & SYSTEMD-NETWORKD ]")
-subprocess.call(["apt", "--autoremove", "purge", "avahi-daemon"])
-subprocess.call(["apt-mark", "hold", "avahi-daemon", "libnss-mdns"])
-subprocess.call(
-    ["ln", "-sf", "/run/systemd/resolve/stub-resolv.conf", "/etc/resolv.conf"])
-subprocess.call(["systemctl",
-                 "enable",
-                 "systemd-networkd.service",
-                 "systemd-resolved.service"])
+    # TODO: confirm with glyph this is supposed to be here
+    print("[ SETTING HOST ]")
```

Comment:
@mycognosist just remembered that I wanted to confirm with you that these three lines are supposed to be here?

before they were in setup_dev_env, 
and I moved them into configure_networking as part of the refactor

Reply:
@mhfowler 

Yes that's correct. `configure_networking` is the best place for them.